region/maridia/inner-pink/Crab Shaft.json

```diff
@@ -1918,6 +1983,45 @@
         "Knocking the crab off while it is on the the door or ceiling will not work."
       ]
     },
+    {
+      "link": [4, 4],
+      "name": "Sciser Farm (1 Sciser, Fast Cycle)",
+      "requires": [
+        {"resetRoom": {"nodes": [3]}},
+        {"or": [
+          "canCrouchJump",
+          "h_underwaterCrouchJumpWithFlashSuit"
+        ]},
+        {"or": [
+          {"and": [
+            "Plasma",
```

Comment:
It feels rare to ever see the crab killed fast enough to not need to jump after it, so I rate this very hard.